_rules/video-only-alternative-for-visual-c3232f.md

```diff
@@ -75,7 +77,7 @@ See [Video Only Element Has Description Track: accessibility support](https://ac
 
 #### Passed Example 1
 
-A video element without audio. The text on the page labels the video as an alternative.
+This `video` element, which has no audio, is a media alternative for the text in the page and labeled as such. (Passes rule [`Video` Element Visual-Only Content Is Media Alternative For Text](https://act-rules.github.io/rules/fd26cf))
```

Comment:
I like that way of giving both infos. Maybe even
```suggestion
This `video` element, which has no audio, is a media alternative for the text in the page and labeled as such. Thus, it passes rule [`Video` Element Visual-Only Content Is Media Alternative For Text](https://act-rules.github.io/rules/fd26cf).
```
(avoids the sentence composed of only a parenthesis)